### PR TITLE
[geolocator] Implement open App/location settings

### PR DESCRIPTION
**Pull request by bbrto21**
Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

File: packages/geolocator/tizen/src/setting.cc [hunk 1]
```diff
@@ -0,0 +1,97 @@
+
```
Comment (bwikbs) [inline packages/geolocator/tizen/src/setting.cc:1]: :scissors:  plz~

**Comment by bwikbs** [comment]
I think ``TizenResult``  can be moved to common part.  (for all tizen plugin)

**Review by bwikbs — APPROVED** [review]
:+1:

File: packages/geolocator/tizen/src/setting.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+#ifndef SETTING_H_
+#define SETTING_H_
+
+#include "tizen_result.h"
+
+namespace Setting {
+
+TizenResult LaunchAppSetting();
+TizenResult LaunchLocatoinSetting();
```
Comment (HakkyuKim) [inline packages/geolocator/tizen/src/setting.h:13]: Just one typo.

```suggestion
TizenResult LaunchLocationSetting();
```

File: packages/geolocator/tizen/src/setting.cc [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  std::string package_name_;
+};
+
+}  // namespace
+
+TizenResult Setting::LaunchAppSetting() {
+  PackageName name;
+  AppControl app_ctrl;
+
+  app_ctrl.SetAppId("com.samsung.clocksetting.apps");
```
[the remark this answers is not in the document]
Comment (bbrto21) [reply packages/geolocator/tizen/src/setting.cc:83]: Yes. this ID came from the setting app, which is installed by default on the Galaxy Watch series.
and the location APIs is not supported on Smart TV profile

File: packages/geolocator/tizen/src/setting.h (hunk 2 above)
Comment (bbrto21) [reply packages/geolocator/tizen/src/setting.h]: Thanks!

**Review by HakkyuKim — APPROVED** [review]
LGTM

**Pull request merged**